tests/console-backend-service/internal/domain/eventing/eventing_test.go

```diff
@@ -12,6 +12,8 @@ import (
 	"github.com/kyma-project/kyma/tests/console-backend-service/internal/module"
 
 	"github.com/pkg/errors"
+	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
```

Comment:
what is it for?

Reply:
It allows to run your the tests locally against cluster running on GCP